rust/tensors.rs

```diff
@@ -81,12 +81,44 @@ impl From<ElementType> for ffi::RuneCoralElementType {
     }
 }
 
+impl From<ffi::RuneCoralElementType> for ElementType {
+    fn from(e: ffi::RuneCoralElementType) -> ElementType {
+        match e {
+            ffi::RuneCoralElementType__NoType => ElementType::NoType,
```

Comment:
Do you know what TensorFlow uses `NoType` for? 

It sounds like this variant exists to act as a marker in case people forget to initialize their `RuneCoralElementType` field. In which case we'd probably want some sort of `fn(ffi::RuneCoralElementType) -> Option<ElementType>` function instead.

Reply:
It is not as simple i think, since models  themselves can come with tensors that use  this enum,  so it is better if we directly expose it as is?